io-ballerina/Module.md

```diff
@@ -0,0 +1,58 @@
+## Module Overview
+
+This module provides file read/write APIs and console print/read APIs. The file APIs allow read and write operations on different kinds of file types such as bytes, text, CSV, JSON, and XML. Further, these file APIs can be categorized as streaming and non-streaming APIs.
+
+The following diagram depicts the overview architecture of the I/O module.
+
+![architecture](./resources/architecture.svg)
+
+The file I/O operations can be further categorized based on serialization and deserialization types such as:
+- Bytes I/O
+- Strings I/O
+- CSV I/O
+- JSON I/O
+- XML I/O
+
```

Comment:
I think we can remove this additional new line.

Reply:
Fixed in here https://github.com/ballerina-platform/module-ballerina-io/pull/179/commits/fd0da0ae53df77cfef77ca08159e7a84a0ea28b6